regolith/schemas.py

```diff
@@ -3703,12 +3707,11 @@
             "required": False,
             "type": "string",
         },
-        # TODO: conditional validation.  If type=colloq or seminar, required is
-        # institution and department, otherwise location
         "location": {
             "description": "city and {state or country} of meeting",
             "required": False,
             "type": "string",
+            "diff_str": "institution"
```

Comment:
The logic is that we have  `meeting_name` and `location` when it is a non-seminar type and we have a `department` and `institution` when it is a seminar type.  What the helpers do is take an input that they will put into either `meeting_name` or `department` depending on type, and an entry that is put in `location` or `institution`, again depending on type.

I am not sure I understand the problem that Priyanka is describing, tbh, and we may not have to fix it even.  Can you understand what the issue is?  Can you look in presentations.yml and see if you can see what the issue is?  Something about columbia in institution and in location?  This is of course fine if one was a seminar at columbia and the other was a conference that was at columbia.....